src/components/Layouts/SidePanelsLayout.tsx

```diff
@@ -126,6 +126,7 @@ export const SidePanelsLayout: FC<SidePanelsLayoutProps> = ({
         style={{
           transform: rightOpen ? 'none' : 'translate(100%)',
         }}
+        id="scrollOverride"
```

Comment:
For the CSS styles we added in [[...contacted]].page. It allows the contact sidebar to scroll when we turn off scrolling on the merge pages.

Reply:
Typically, it's best to use classes for CSS instead of IDs, as IDs hold a lot more value when classes are used for JS.
You don't have to change here, though.